pkg/deploy/controller/deploymentconfig/controller.go

```diff
@@ -40,9 +40,9 @@ type fatalError string
 // transientError is an error which should always be retried (indefinitely).
 type transientError string
 
-func (e fatalError) Error() string { return "fatal error handling deploymentConfig: " + string(e) }
+func (e fatalError) Error() string { return "fatal error handling deployment config: " + string(e) }
```

Comment:
`fmt.Errorf` returns error values and here we need `fatalError` to implement the error interface so needs to have an `Error() string` method.